setup/redhat/postinstall.sh

```diff
@@ -10,6 +10,10 @@ ODOO_LOG_DIR=/var/log/odoo
 ODOO_LOG_FILE=$ODOO_LOG_DIR/odoo-server.log
 ODOO_USER="odoo"
 
+if [ -d /usr/lib/python3.7 ]; then
+    ln -s /usr/lib/python3.6/site-packages/odoo /usr/lib/python3.7/site-packages/odoo
+fi
```

Comment:
@xmo-odoo what do you think of this ugly hack that could fix odoo/odoo#30579.
Do you think of any bad things that could happen ?

Reply:
Only question I'd have is do we know for sure the site-packages directory exists on the target?

Reply:
@xmo-odoo now site-package directory is created when needed